server/__test__/mock_data/mock_users.js

```diff
@@ -11,12 +11,12 @@ const mockUser = {
 
   wrongEmail: {
     email: 'wrongEmail@com',
-    password: 'password'
+    password: 'PassWord123..'
   },
 
   unknownUser: {
     email: 'unknown_person@gmail.com',
-    password: 'password'
+    password: 'PassWord123..'
```

Comment:
is there a reason why you changed all passwords to password123?

Reply:
I increased password acceptance criteria. It only accepts password that has uppercase letter, lowercase and at least a number